src/canisters/frontend/ic-certified-assets/src/state_machine.rs

```diff
@@ -574,17 +613,28 @@ impl From<StableState> for State {
             ..Self::default()
         };
 
-        for (asset_name, asset) in state.assets.iter_mut() {
-            for enc in asset.encodings.values_mut() {
-                enc.certified = false;
+        let assets_keys: Vec<_> = state.assets.keys().map(|key| key.clone()).collect();
+        for key in assets_keys {
```

Comment:
```suggestion
        state.assets.keys().for_each(|key| {
```

Reply:
Sadly, this doesn't work either because of the `assets.get_mut` two lines below

Reply:
Ah, I thought it might work since it compiles and doesn't move/delete/insert any assets.